components/reusable-components/navigation/Navigation.tsx

```diff
@@ -1,36 +1,72 @@
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+
 'use client';
 
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+
+// eslint-disable-next-line import/no-duplicates
 import Link from 'next/link';
+import Image from 'next/image';
 import styles from './navigation.module.scss';
 import Button from '../button/Button';
+import { useTheme } from '../../context/ThemeContext';
+// eslint-disable-next-line import/no-unresolved
+import solImage from '../../../public/sol.png';
+// eslint-disable-next-line import/no-unresolved
+import moonImage from '../../../public/lune.png';
 
 const Navigation = () => {
+  const { theme, toggleTheme } = useTheme();
+  const [isSun, setIsSun] = useState(true);
+
+  useEffect(() => {
+    document.body.className = theme;
+  }, [theme]);
+  useEffect(() => {
+    const storedIsSun = localStorage.getItem('isSun');
+    if (storedIsSun !== null) {
+      setIsSun(storedIsSun === 'true');
+    }
+  }, []);
+  const handleClick = () => {
+    toggleTheme();
+    setIsSun((prevIsSun) => {
+      const newIsSun = !prevIsSun;
+      localStorage.setItem('isSun', String(newIsSun));
+      return newIsSun;
+    });
+  };
+
   return (
     <nav className={styles.largeNavbar}>
       <div className={styles.navContainer}>
-        <div className={styles.leftContent}>
-          <Link className={`display-s ${styles.navigationLogo}`} href="/">
-            LearnHub.mk
-          </Link>
-          <div id="navbarNav">
-            <ul className={styles.menuElementsNav}>
-              <Link className={`headline-s ${styles.menuElements}`} href="/">
-                Визија{' '}
-              </Link>
-              <Link className={`headline-s ${styles.menuElements}`} href="/">
-                {' '}
-                Цели{' '}
-              </Link>
-              <Link className={`headline-s ${styles.menuElements}`} href="/">
-                {' '}
-                Контакт{' '}
-              </Link>
-              <Link className={`headline-s ${styles.menuElements}`} href="/">
-                Блог{' '}
-              </Link>
-            </ul>
-          </div>
+        <Link className={`display-s ${styles.navigationLogo}`} href="/">
+          <Image
+            src="/logo/Horizontal-Logo_Full-Color.svg"
+            className={`${styles.navigationLogo}`}
+            alt="LearnHub Logo"
+            width={250}
+            height={55}
+          />
+        </Link>
```

Comment:
We should toggle the logo as well.
If the theme is dark we should display the logo with white letters.
If the theme is light we show the logo you have placed here.